plans/artifactory-pro/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo "Starting Artifactory Pro"
+
+export JAVA_HOME=$(hab pkg path core/server-jre)
+export ARTIFACTORY_HOME={{pkg.svc_var_path}}
+
+{{pkg.svc_var_path}}/bin/artifactory.sh 2>&1
```

Comment:
I'm honestly not sure which is better than the other. Certainly having a separate run hook is less long. :)